### PR TITLE
quote.py for Quotes

### PR DESCRIPTION
**Pull request by Divyanshu29**
Created Plugin for Quote Extraction of Trailer Park Boys.

### CODE REVIEW

File: honeybot/quote.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+        quotes=[]
+
+        for i in bs.find_all('dl'):
+            quotes.append(i.text)
+        for i in range(int(sys.argv[1])):
+            print random.choice(quotes)
+            print ("--------------------------------------------------------------------------")
+   
+        except:
```
Comment (wupewu) [inline honeybot/quote.py]: This line don't have sense because bot will not work without internet connection

File: honeybot/quote.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        #try:
+        data = urllib.urlopen("https://en.wikiquote.org/wiki/Trailer_Park_Boys").read()
+        bs = BeautifulSoup(data,"lxml")
+
+        quotes=[]
+
+        for i in bs.find_all('dl'):
+            quotes.append(i.text)
+        for i in range(int(sys.argv[1])):
+            print random.choice(quotes)
```
Comment (wupewu) [inline honeybot/quote.py]: https://github.com/pyhoneybot/honeybot/blob/master/honeybot/plugins/joke.py#L78
Check how to send messages as bot.
Print line will appear in console/logs, but not in chat.

File: honeybot/quote.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        pass
+    
+    def run():
+        #try:
+        data = urllib.urlopen("https://en.wikiquote.org/wiki/Trailer_Park_Boys").read()
+        bs = BeautifulSoup(data,"lxml")
+
+        quotes=[]
+
+        for i in bs.find_all('dl'):
```
Comment (wupewu) [inline honeybot/quote.py]: Consider changing 'i' variable name to 'quote' for easier reading.

File: honeybot/quote.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+[About]
+Extracts quotes and send random quote from Trailer Park Boys
+
+[Commands]
+>>>.Quote <count>
+"""
+from bs4 import BeautifulSoup
+import urllib
+import random
+import sys
```
Comment (wupewu) [inline honeybot/quote.py]: Unnecessary import since we won't be parsing any console arguments

File: honeybot/quote.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+class Plugin:
+
+    def __init__(self):
+        pass
+    
+    def run():
+        #try:
```
Comment (wupewu) [inline honeybot/quote.py]: Commented out try-except won't work

File: honeybot/quote.py (hunk 1 above)
Comment (wupewu) [inline honeybot/quote.py]: Try to catch specific exceptions (check urllib docs)

**Comment by Divyanshu29** [comment]
Done

**Pull request merged**